src/annotation/band.js

```diff
@@ -55,7 +55,11 @@
                     });
 
                 g.select('path')
```

Comment:
This problem is rife throughout our use of the `fc.svg.* ` components e.g. [bar series](https://github.com/ScottLogic/d3fc/blob/master/src/series/bar.js#L45). I do think your changes represent the pattern we should be following though, should we create an issue to fix the rest?